packages/ra-ui-materialui/src/list/filter/FilterButton.stories.tsx

```diff
@@ -372,3 +372,45 @@ export const WithAutoCompleteArrayInput = (args: {
         </MemoryRouter>
     );
 };
+
+export const WithMuiButtonProps = () => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Could  it be interesting to have 2 stories to show it: `size` and `variant`?

Reply:
Done